test/sharness/t0113-gateway-symlink/README.md

```diff
@@ -0,0 +1,16 @@
+# Dataset description/sources
+
+- testfiles.car
+  - raw CARv1
+
+generated with:
+
+```sh
+mkdir testfiles &&
+echo "content" > testfiles/foo &&
+ln -s foo testfiles/bar &&
+ROOT_DIR_CID=$(ipfs add -Qr testfiles) &&
+ipfs dag export $ROOT_DIR_CID > testfiles.car
+
+# ROOT_DIR_CID=QmWvY6FaqFMS89YAQ9NAPjVP4WZKA1qbHbicc9HeSKQTgt
+```
```

Comment:
Did you start working on the part of https://github.com/ipfs/gateway-conformance with which we could codify car creation part?

Reply:
Or is it the next step in which we'll also ensure that we can generate cars that are exactly the same as here?

Reply:
Or did we drop this idea?

Reply:
> Or is it the next step in which we'll also ensure that we can generate cars that are exactly the same as here?

Opposite, we want to replace fixtures generated at runtime with static, deterministic car files. These files would come with the specs (or the test definition). @lidel shared an example with the _redirects spec in https://github.com/ipfs/gateway-conformance/issues/2#issuecomment-1428982796.

It doesn't change much to the gateway-conformance repo: we replace vanilla files and folders (like: https://github.com/ipfs/gateway-conformance/tree/main/fixtures/dir) with car files.

Reply:
What I'm getting at is that in my opinion we need one more step - tooling which can do the following for us:
```
mkdir testfiles &&
echo "content" > testfiles/foo &&
ln -s foo testfiles/bar &&
ROOT_DIR_CID=$(ipfs add -Qr testfiles) &&
ipfs dag export $ROOT_DIR_CID > testfiles.car
```

I'm not advocating doing it at runtime. I think we'd benefit from having an easy-path for creating a car file. Sort of fixture-CAR-as-code ;) And I do agree that it should be separate from the test code.

Reply:
Discussed this during 1:1, came to agreement that it's important to capture "how" CARs come to be but we want to address this later once we learn all the requirements we might have for CAR creation.